docs/pages/blog/introducing-pigment-css.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Introducing Pigment CSS: the next generation of CSS-in-JS'
+title: 'Introducing the first Pigment CSS alpha release'
```

Comment:
I think we should emphasis that it's in alpha.

Reply:
The original title rings much better than this one. In my opinion, we need first to introduce the product/tool and then say which version it is available at, which is a detail. Plus, just from this title, you don't know what Pigment is, whereas with the original one, we not only know that it is a CSS-in-JS library, but we also know that it is positioning itself as a next-gen, a strategic point.